data/ext/pending/issue-3563-examples.txt

```diff
@@ -0,0 +1,134 @@
+TYPES: #eg-0489 Certification
+
+PRE-MARKUP:
+
```

Comment:
I would not have this generic text, but have instead a textual description of what is represented in the json-ld, i.e. that there a three programs, with the advantages and benefits in plain english.

Reply:
Fixed the TYPES and updated the plain text version under "PRE-MARKUP"